packages/mobile/src/identity/verification.ts

```diff
@@ -391,6 +401,7 @@ function attestationCodeReceiver(
       yield put(inputAttestationCode({ code, issuer }))
     } catch (error) {
       Logger.error(TAG + '@attestationCodeReceiver', 'Error processing attestation code', error)
+      CeloAnalytics.track(CustomEventNames.verification_error, { error })
```

Comment:
I think it would be nice to get some events when we receive the code ahead as well